Desktop/Regression_Test_Plan_Minor_Release.md

```diff
@@ -212,103 +212,103 @@ TestID | Test Case | Steps to reprouce| Expected Result | Result | Related Comme
-2a :robot: | Linux+Mac: Check that in the setup options the checkbox is enable to sync all the account by default | 1. Start to setup oC Desktop 2. Fill in server, user and password 3. Click on "Choose what to sync" | At window is opened with the directory tree |:heavy_check_mark:  | tst_syncing |
+TestID | Test Case | Steps to reprouce| Expected Result | Result | Related Comment | Server
+------------ | ------------- | -------------- | ----- | ------ | ------ | -----
+1  :robot: | Check that in the setup options after fill in the server and the user you have to choose the server and local folder to sync| 1. Start to setupo oC Desktop 2. Fill in server, user and password 3. The server and local folder to sync is shown|1. On server (you have to choose what do yo want to sync) 2. local folder (the local folder where the data are sync) | :heavy_check_mark: | tst_syncing | :robot: oC10,<br> :robot: oCIS |
+2a | Linux+Mac: Check that in the setup options the checkbox is enable to sync all the account by default | 1. Start to setup oC Desktop 2. Fill in server, user and password 3. Click on "Choose what to sync" | At window is opened with the directory tree |:heavy_check_mark:  | tst_syncing | :robot: oC10,<br> :robot: oCIS | 
 2b | Win10: Check that in the setup options the checkbox is enable to "use virtual files" default | 1. Start to setup oC Desktop 2. Fill in server, user and password 3. Click on "Synchronize everything from server"; 4. "Choose what to sync" | At window is opened with the directory tree | :construction: |  |
-3 :robot: | Check that all files and folders are synced when you select to sync all | 1. Start to setup oC Desktop 2. Fill in server, user and pass 3. Click on "Sync everything from server" and wait until everything is sync| Everything is synced | :heavy_check_mark: | tst_syncing |
-4 :robot: | Verify that you can select to sync only one folder | 1. Start to setup the oC Desktop 2. The new window select what folder sync is shown 3. Select to sync only one folder 4. Click on OK | Only one folder is sync |:heavy_check_mark:  | tst_syncing |
-5 :robot: | Verify that the folder to select are sorted by name/size| 1. Start to setup the oC Desktop 2. The new window to select what folder sync is shown 3. Select all the folder except one of them| Everything except one folder is synced  |:heavy_check_mark:  | tst_syncing |
-6 :robot: | Check that there are another option: skip sync, so that nothing is synced | 1. Start to setup oC Desktop 2. Fill in server, user and password 3. Click on "Manually create folder sync connection"| The settings windows is openend but no local folder is configured| :heavy_check_mark: | tst_syncing |
+3 :robot: | Check that all files and folders are synced when you select to sync all | 1. Start to setup oC Desktop 2. Fill in server, user and pass 3. Click on "Sync everything from server" and wait until everything is sync| Everything is synced | :heavy_check_mark: | tst_syncing | :robot: oC10 |
```

Comment:
Should be covered by https://github.com/owncloud/client/blob/0fbaf515aa67e3265a2309ed2033061af53f5e6e/test/gui/tst_syncing/test.feature#L21
```suggestion
3 :robot: | Check that all files and folders are synced when you select to sync all | 1. Start to setup oC Desktop 2. Fill in server, user and pass 3. Click on "Sync everything from server" and wait until everything is sync| Everything is synced | :heavy_check_mark: | tst_syncing | :robot: oC10,<br> :robot: oCIS |
```